app_layout.py

```diff
@@ -5,15 +5,16 @@
 from pathlib import Path
 
 
-def instructions():
-    """Instructions for using app"""
-    path_instructions = (
-        Path(__file__).parents[1]
+def open_txt_doc(filename):
+    """Opens texts in docs folder that go in front end of app"""
+    path = (
+        Path(__file__).parent
         / 'docs'
-        / 'instructions.txt'
+        / 'app'
+        / filename
```

Comment:
Does this path need to take up multiple lines? Only do so if it runs over the line length req, or is causing significant unreadability issues (which I don't think this is).